docs/pages/operations-guide.md

```diff
@@ -371,361 +383,285 @@ Given the above configuration, let's assume `10.0.10.8` is being decommissioned
+The instructions below describe how you can update the configuration for a running DC/OS service.
 
-Either or both of these lists may be useful depending on the context. Click on the `Sandbox` link for one of these tasks and then start looking at sandbox content. Files named `stderr` and `stdout` hold logs produced both by the SDK Executor process (a small wrapper around the service task) as well as any logs produced by the task itself. These files are automatically paginated at 2MB increments, so older logs may also be examined until they are automatically pruned. For an example of this behavior, see the [scheduler sandbox](img/ops-guide-scheduler-sandbox.png) linked earlier.
+##### Preparing configuration
 
-[<img src="img/ops-guide-task-sandbox.png" alt="contents of a task sandbox" width="400"/>](img/ops-guide-task-sandbox.png)
+If you installed this service with Enterprise DC/OS 1.10, you can fetch the full configuration of a service (including any default values were applied during installation). For example, for `dse`:
 
-### Mesos Agent logs
+```bash
+$ dcos dse describe > options.json
+```
 
-Occasionally, it can also be useful to examine what a given Mesos agent is doing. The Mesos Agent handles deployment of Mesos tasks to a given physical system in the cluster. One Mesos Agent runs on each system. These logs can be useful for determining if there's a problem at the system level that is causing alerts across multiple services on that system.
+Make any configuration changes to this `options.json` file. 
 
-Navigate to the agent you want to view either directly from a task by clicking the "Agent" item in the breadcrumb when viewing a task (this will go directly to the agent hosting the task), or by navigating through the "Agents" menu item at the top of the screen (you will need to select the desired agent from the list).
+If you installed this service with a prior version of DC/OS, this configuration will not have been persisted by the the DC/OS package manager. You can instead use the `options.json` file that was used when [installing the service](#initial-service-configuration). 
 
-In the Agent view, you'll see a list of frameworks with a presence on that Agent. In the left pane you'll see a plain link named "LOG". Click that link to view the agent logs.
+<strong>Note:</strong> You need to specify all configuration values in the `options.json` file when performing a configuration update. Any unspecified values will be reverted to the default values specified by the DC/OS service. See the "Recreating `options.json`" section below for information on recovering these values.
 
-[<img src="img/ops-guide-agent.png" alt="view of tasks running on a given agent" width="400"/>](img/ops-guide-agent.png)
+###### Recreating `options.json` (optional)
 
-### Logs via the CLI
+If the `options.json` from when the service was last installed or updated is not available, you will need to manually recreate it using the following steps.
```

Comment:
This would occur if you had installed the service with 1.9 and then later upgraded to 1.10 while leaving the service in-place